Dapper.Tests.Contrib/Dapper.Tests.Contrib.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="Microsoft.Data.Sqlite" Version="1.1.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
     <PackageReference Include="MySqlConnector" Version="0.44.1" />
-    <PackageReference Include="System.Data.SqlClient" Version="4.4.0" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.6.0" />

```

Comment:
Hmmm, we shouldn't actually need this ref in anything with a Dapper project ref - should only need it on Dapper and Dapper.StrongName, right? And delete the other lines here?

Reply:
You're right - pushed fix